WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.m

```diff
@@ -317,6 +317,9 @@ - (void)showDetailsForNoteWithID:(NSString *)notificationID
     if (notification) {
         DDLogInfo(@"Pushing Notification Details for: [%@]", notificationID);
         
+        NSDictionary *properties = @{ @"type" : (notification.type ?: [NSString string]) };
+        [WPAnalytics track:WPAnalyticsStatPushNotificationAlertPressed withProperties:properties];
```

Comment:
Type is expected to be always there (this is just a failsafe). But perhaps it makes more sense to move the check outside. Updating, thanks!